app/vmselect/main.go

```diff
@@ -700,7 +693,6 @@ var (
 	expandWithExprsRequests = metrics.NewCounter(`vm_http_requests_total{path="/expand-with-exprs"}`)
 	prettifyQueryRequests   = metrics.NewCounter(`vm_http_requests_total{path="/prettify-query"}`)
 
-	vmalertRequests   = metrics.NewCounter(`vm_http_requests_total{path="/vmalert"}`)
```

Comment:
This metric is still used by `/vmalert` prefix requests.

Reply:
fixed